scripts/foundry/contract-upgrades/2.1.0/DeployNitroContracts2Point1Point0UpgradeAction.s.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import {DeploymentHelpersScript} from "../../helper/DeploymentHelpers.s.sol";
+import {
+    NitroContracts2Point1Point0UpgradeAction,
+    IOneStepProofEntry
+} from "../../../../contracts/parent-chain/contract-upgrades/NitroContracts2Point1Point0UpgradeAction.sol";
+import {MockArbSys} from "../../helper/MockArbSys.sol";
+
+/**
+ * @title DeployNitroContracts2Point1Point0UpgradeActionScript
+ * @notice This script deploys OSPs, ChallengeManager and Rollup templates, and the upgrade action.
+ */
+contract DeployNitroContracts2Point1Point0UpgradeActionScript is DeploymentHelpersScript {
+    // ArbOS v31 https://github.com/OffchainLabs/nitro/releases/tag/consensus-v31
+    bytes32 public constant WASM_MODULE_ROOT = 0x260f5fa5c3176a856893642e149cf128b5a8de9f828afec8d11184415dd8dc69;
+
+    // ArbOS v20 https://github.com/OffchainLabs/nitro/releases/tag/consensus-v20
+    bytes32 public constant COND_WASM_MODULE_ROOT = 0x8b104a2e80ac6165dc58b9048de12f301d70b02a0ab51396c22b4b4b802a16a4;
+
+    function run() public {
+        bool isArbitrum = vm.envBool("PARENT_CHAIN_IS_ARBITRUM");
+        if (isArbitrum) {
+            // etch a mock ArbSys contract so that foundry simulate it nicely
+            bytes memory mockArbSysCode = address(new MockArbSys()).code;
+            vm.etch(address(100), mockArbSysCode);
+        }
```

Comment:
remove this

Reply:
https://github.com/OffchainLabs/orbit-actions/pull/16/commits/5fd1e7e02307396e5da2a08acabfb0ff1de8b619